### ISSUE TITLE
Missing tls Key when convert vmess to clash.

### ISSUE THREAD

**Issue by CokeMine** [tindy2013/subconverter#327]
Missing tls Key when convert vmess to clash.

The decoded vmess link is
```
{
"v": "2",
"ps": "XXXX",
"add": "XXXX",
"port": "443",
"id": "XXXX",
"aid": "XXXX",
"net": "ws",
"type": "none",
"host": "XXXX",
"path": "/XXXX",
"tls": "tls"
}
```

with a tls key, value is "tls".

But converted one will be {tls: false, skip-cert-verify: false}.

The tls mode was disabled.

The version is the latest subconverter docker version.

**Comment by tindy2013** [comment]
Fixed in commit 3afe9986bea32abb3b2a3415bf3018ecb01a5bb4

**Comment by CokeMine** [comment]
Works well, thank you!
